xml/operations-maintenance-upgrade_maintenance.xml

```diff
@@ -0,0 +1,1828 @@
+ansible-playbook -i hosts/verb_hosts ardana-reboot.yml --limit ardana-cp-mml-m2
+ansible-playbook -i hosts/verb_hosts ardana-reboot.yml --limit ardana-cp-mml-m3
+</screen>
+      <warning>
+        <para>
+          Do not reboot all your control plane nodes at the same time.
+        </para>
+      </warning>
+     </step>
+     <step>
+       <para>
+         To reboot a specific control plane node, you can use the above
+         ansible-playbook commands as follows:
+       </para>
+       <screen>&prompt.ardana;cd ~/scratch/ansible/next/ardana/ansible
+&prompt.ardana;ansible-playbook -i hosts/verb_hosts ardana-reboot.yml --limit ardana-cp-mml-m3
+
+PLAY [all] ********************************************************************
+
+TASK: [setup] *****************************************************************
+ok: [ardana-cp-mml-m3]
+
+PLAY [localhost] **************************************************************
+
+TASK: [pbstart.yml pb_start_playbook] *****************************************
+ok: [localhost] => {
+    "msg": "Playbook started - ardana-reboot.yml"
+}
+
+msg: Playbook started - ardana-reboot.yml
+
+
+
+...
+
+PLAY [localhost] **************************************************************
+
+TASK: [pbfinish.yml pb_finish_playbook] ***************************************
+ok: [localhost] => {
+    "msg": "Playbook finished - ardana-status.yml"
+}
+
+msg: Playbook finished - ardana-status.yml
+
+PLAY [localhost] **************************************************************
+
+TASK: [pbfinish.yml pb_finish_playbook] ***************************************
+ok: [localhost] => {
+    "msg": "Playbook finished - ardana-reboot.yml"
+}
+
+msg: Playbook finished - ardana-reboot.yml
+
+PLAY RECAP ********************************************************************
+ardana-cp-mml-m3           : ok=389  changed=105  unreachable=0    failed=0
+localhost                  : ok=27   changed=1    unreachable=0    failed=0
+</screen>
+     </step>
+   </procedure>
+   <note>
+     <para>
+       You can reboot more than one control plane node at a time, but only if
+       they are members of different control plane clusters. For example, you
+       could reboot one node out of each of the Openstack controller, database,
+       Swift, monitoring or logging clusters, so long as doing do only reboots
+       one node out of each cluster at the same time.
+     </para>
+   </note>
+
+   <para>
+     When rebooting the first member of the control plane cluster where
+     monitoring services run, the <literal>monasca-thresh</literal> service can sometimes fail
+     to start up in a timely fashion when the node is coming back up after
+     being rebooted. This can cause <filename>ardana-reboot.yml</filename> to fail.
+     See below for suggestions on how to handle this problem.
+   </para>
+
+   <procedure>
+     <title>Getting <literal>monasca-thresh</literal> Running After an <filename>ardana-reboot.yml</filename> Failure</title>
```

Comment:
I am not sure [literal] is the best tag in this paragraph...

Reply:
Please make a suggestion, in that case

Reply:
Apologies. I was making assumptions. Previously, at line 1256 (when "monasca" is first discussed), we use:

> (for example, the &lt;systemitem class="service"&gt;monasca-thresh&lt;/systemitem&gt; service)

It is wordier but I think it is more formally correct. A search-and-replace could probably achieve this.